bin/bugsnag-dsym-upload

```diff
@@ -13,18 +13,22 @@
 function print_usage() {
     echo "Usage: $0 [--symbol-maps DIR] dSYMS_PATH"
     echo
-    echo "-h, --help          Displays this message"
-    echo "-v, --verbose       Print verbose logging output during execution"
-    echo "--api-key API_KEY   The API key of the project the dSYM should be applied to"
-    echo "--symbol-maps DIR   Path to a directory of bitcode symbol maps. The"
-    echo "                    dSYM files will be restored with symbols prior to"
-    echo "                    upload. Requires dsymutil."
-    echo "--upload-server URL The server receiving dSYM files. Set this value if"
-    echo "                    using an on-premise Bugsnag installation"
-    echo "--project-root DIR  The root directory of the project. This will help to"
-    echo "                    group error reports by project"
-    echo "dSYMS_PATH          A directory or .zip file containing *.dSYM bundles to"
-    echo "                    upload"
+    echo "-h, --help                 Displays this message"
+    echo "-v, --verbose              Print verbose logging output during execution"
+    echo "-m, --allow-missing-dwarf  Throw warnings instead of errors when a dSYM with missing"
+    echo "                           DWARF data is found"
```

Comment:
These two options both feel like "ignore" options, so I would suggest we have `--ignore-missing-dwarf` for consistency.

Reply:
allow changed for ignore [`aa238a2` (#60)](https://github.com/bugsnag/bugsnag-dsym-upload/pull/60/commits/aa238a217d45bb9e48ae81022ff0d3f5623f89e9)